src/Controllers/SettingsController.php

```diff
@@ -243,10 +243,16 @@ public function settingsMenu(): array
         $menu = [
             url('/settings/profile')  => 'settings.profile',
             url('/settings/password') => 'settings.password',
-            url('/settings/language') => 'settings.language',
-            url('/settings/theme')    => 'settings.theme',
         ];
 
+        if (count(config('locales')) > 1) {
+            $menu[url('/settings/language')] = 'settings.language';
+        }
+
+        if (count(config('themes')) >= 2) {
```

Comment:
Here too (as its even more confusing that way) ;)

Reply:
fixed